examples/css2d_label.html

```diff
@@ -218,10 +225,14 @@
 
 				gui = new GUI();
 
-				gui.add( layers, 'Toggle Name' );
-				gui.add( layers, 'Toggle Mass' );
-				gui.add( layers, 'Enable All' );
-				gui.add( layers, 'Disable All' );
+				const folder = gui.addFolder( 'Camera Layers' );
```

Comment:
Instead of having an empty folder `Controls` and then `Camera Layers`, try it like this:
```js
gui = new GUI();
gui.title( 'Camera Layers' );

gui.add( layers, 'Toggle Name' );
gui.add( layers, 'Toggle Mass' );
gui.add( layers, 'Enable All' );
gui.add( layers, 'Disable All' );

gui.open();
```